src/inverseRobot.js

```diff
@@ -8,6 +8,19 @@
 
 function inverseRobot(robot) {
   // write code here
+  const inversedRobot = {};
+
+  for (const key in robot) {
+    const newKey = robot[key];
+
+    if (inversedRobot[newKey]) {
```

Comment:
The condition for checking if a newKey already exists in inversedRobot is not accurate. It will fail if the newKey is '0' or any other falsy value. It is recommended to use 'hasOwnProperty' method to check if a property exists in an object.